fly.toml

```diff
@@ -12,6 +12,9 @@ auto_rollback = true
 source = "data"
 destination = "/data"
 
+[env]
+# SENTRY_DSN =
```

Comment:
Sentry DSNs are not sensitive so I think it make more sense to set it here rather than via CLI. But this makes it a bit different so I'm definitely open to other strategies.

Reply:
I would rather do stuff like this in the dockerfile, but even this I think is better as part of the env file so it can be changed more easily.

Reply:
@kentcdodds Hmm so the problem with doing it in a `.env` file is that it won't ship during a deployment. How do we get around having to duplicate your environment variables since we need it during the Fly deployments?

Reply:
> having to duplicate your environment variables 

I'm not sure I follow. Why do we need to duplicate? We just set it once like we do with the SESSION_SECRET etc. and then we don't worry about it anymore.

Reply:
@kentcdodds Missed this comment earlier. But if you look at `SESSION_SECRET`, you'll see that it exits in the `.env` file and there's also a separate step for `fly secrets set`. But I think that's unavoidable.

Reply:
Oh, I see what you mean. Yeah, the `.env` is just generated anyway because you don't run with sentry enabled locally anyway.